internal/utils.go

```diff
@@ -486,3 +486,19 @@ func verifyManifestPath(baseDirectory string, manifestPath string) error {
 
 	return nil
 }
+
+// Check policy-templates to see if all the remediation actions match, if so set the root policy remediation action
+func getRootRemediationAction(policyTemplates []map[string]map[string]interface{}) string {
+	var action string
+
+	for _, value := range policyTemplates {
+		spec := value["objectDefinition"]["spec"].(map[string]interface{})
```

Comment:
In the case where an `IAMPolicy` or `CertificatePolicy` is specified as a manifest, then `policy-template` entry is not generated and thus not guaranteed to have `spec` and `spec.remediationAction` set, so you need to account for bad user input here. You can do this by verifying the type assertions.